docs/getting-started-guides/fedora/flannel_multi_node_cluster.md

```diff
@@ -0,0 +1,219 @@
+
+**Perform following commands on the kubernetes master** 
+
+* Configure flannel by creating a `flannel-config.json` in your current directory on fed-master. flannel provides udp and vxlan among other overlay networking backend options. In this guide, we choose kernel based vxlan backend. The contents of the json are:
+
+```
+{
+    "Network": "18.16.0.0/16",
+    "SubnetLen": 24,
+    "Backend": {
+        "Type": "vxlan",
+        "VNI": 1
+     }
+}
+```
+**NOTE:** Choose an IP range that is *NOT* part of the public IP address range.
+
+* Add the configuration to the etcd server on fed-master.
+
+```
+# curl -L http://fed-master:4001/v2/keys/coreos.com/network/config -XPUT --data-urlencode value@flannel-config.json
+```
+
+* The output should look like:
+
+```
+{"action":"set","node":{"key":"/coreos.com/network/config","value":"{\n    \"Network\": \"18.16.0.0/16\",\n    \"SubnetLen\": 24,\n    \"Backend\": {\n        \"Type\": \"vxlan\",\n        \"VNI\": 1\n     }\n}\n","modifiedIndex":3,"createdIndex":8}}
+```
+
+* Verify the key exists in the etcd server on fed-master.
+
+```
+# curl -L http://fed-master:4001/v2/keys/coreos.com/network/config
+{"action":"get","node":{"key":"/coreos.com/network/config","value":"{\n    \"Network\": \"18.16.0.0/16\",\n    \"SubnetLen\": 24,\n    \"Backend\": {\n        \"Type\": \"vxlan\",\n        \"VNI\": 1\n     }\n}\n","modifiedIndex":8,"createdIndex":8}}
+```
+
+**Perform following commands on all kubernetes nodes** 
+
+* Edit the flannel configuration file /etc/sysconfig/flanneld as follows:
+
+```
+# Flanneld configuration options
+
+# etcd url location.  Point this to the server where etcd runs
+FLANNEL_ETCD="http://fed-master:4001"
+
+# etcd config key.  This is the configuration key that flannel queries
+# For address range assignment
+FLANNEL_ETCD_KEY="/coreos.com/network"
+
+# Any additional options that you want to pass
+FLANNEL_OPTIONS="eth0"
+```
+
+* Note: The network interface (capable of communicating with other nodes) is commonly `eth0` but might be something else (`ens3`). Use `ip a` to list network interfaces.
+
+* Enable the flanneld service and reboot.
+
+```
+# systemctl enable flanneld
+# systemctl reboot
```

Comment:
you're right that it isn't clear if / when docker should be started. i was assuming a fresh system, and noticing that docker instructions were absent.

at the top there could be a note: if you have docker running, stop it and ip link delete docker0. then later the instructions can mention starting it again, after flannel and thus w/ the proper configs.